lib/lita/handlers/totems.rb

```diff
@@ -171,6 +172,11 @@ def add(response)
             # queue:
             queue_size = redis.rpush("totem/#{totem}/list", user_id)
             redis.hset("totem/#{totem}/waiting_since", user_id, Time.now.to_i)
+            current_owner_id = redis.get("totem/#{totem}/owning_user_id")
+            unless redis.sismember("user/#{current_owner_id}/totems/reminder", totem)
+              robot.send_messages(Lita::Source.new(user: Lita::User.find_by_id(current_owner_id)), %{#{response.user.name} just added the "#{totem}" totem, are you still using it?})
```

Comment:
[question] Is find by id always safe to use? Was curious about the bug involving kicking new users and if operations involving new users are somewhere unsafe and need a fallback?

Reply:
> [question] Is find by id always safe to use? Was curious about the bug involving kicking new users and if operations involving new users are somewhere unsafe and need a fallback?

we shouldn't have any issues using find_by_id(). The problem we had with kicking new users was because we used a deprecated method ([ref](https://github.com/change/lita-slack/pull/3)), why it complains only on the kick command? not completely sure, might be something related to what slack has on their website 
"We'll stop allowing newly created Slack apps to use these deprecated APIs beginning June 10th, 2020" but either way, once we upgraded to use the new method we are able to kick users with the kick command

Reply:
GTK. Thanks, @msavir +